src/constants.js

```diff
@@ -0,0 +1,45 @@
+// GitHub and GitHub Action Constants
```

Comment:
I think if you add `// @flow` to the top of this file that'll fix the flow errors you see below.